src/components/ec-full-screen-overlay/ec-full-screen-overlay.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <transition name="ec-full-screen-overlay__fade">
+    <div
+      v-show="show"
```

Comment:
Even if `v-show` does the job I have the feeling v-if is better for this particular case.
The reason is `v-show` will keep the element always in the DOM and just toggle its visibility, on the other hand `v-if` will completely destroy it so when we do not need it.